src/pages/components/select/usage.mdx

```diff
@@ -211,17 +211,21 @@ Inline selects have less visual weight on a page because they are borderless.
 
 1. **Labels:** Text that informs the user what to expect in the list of dropdown
    options.
-2. **Default option:** Usually a frequent option that users would choose from a
-   select list.
+2. **Default option:** An empty option selected by default. The user can decide
```

Comment:
Should the # 2  "Default option" label be called "Selected option" instead? In the image example above, it is just a little confusing because there is already something selected from the list, and it isn't an empty selection which is the default behavior.